parsel/selector.py

```diff
@@ -1,5 +1,5 @@
 """
-XPath selectors based on lxml
+JPath/XPath selectors based on lxml and jmespath
```

Comment:
I don’t think we should use “JPath” as a short form for “JMESPath”, it may be confusing, specially given https://pypi.org/project/jpath/ exists. Same goes for the implemented method name.

Reply:
how about jsonpath? jmespath is difficult to remembered, especially for a man who first language is not english. most developer in the world can remember simple English easily.

Reply:
It has been changed to jsonpath but it will look too long when chained. If people think jsonpath is OK, I am ready to submit it. Do you have any idea?

example:
```
>>> sel.xpath('//div/content').jsonpath('user[*].name').getall()
```